base_layer/core/src/mempool/config.rs

```diff
@@ -46,12 +48,18 @@ impl SubConfigPath for MempoolConfig {
 #[serde(deny_unknown_fields)]
 pub struct MempoolServiceConfig {
     /// Number of peers from which to initiate a sync. Once this many peers have successfully synced, this node will
-    /// not initiate any more mempool syncs. Default: 2
+    /// not initiate any more mempool syncs.
     pub initial_sync_num_peers: usize,
-    /// The maximum number of transactions to sync in a single sync session Default: 10_000
+    /// The maximum number of transactions to sync in a single sync session
     pub initial_sync_max_transactions: usize,
     /// The maximum number of blocks added via sync or re-org to triggering a sync
     pub block_sync_trigger: usize,
+    /// The maximum number of transactions a peer can specifically request.
+    pub max_request_transactions: usize,
+    /// The length of time between checking and, if required, requesting the transactions from the want list.
+    pub request_want_list_interval: Duration,
+    /// Maximum concurrent inbound handler tasks.
+    pub max_concurrent_inbound_tasks: usize,
```

Comment:
Please add to `c_base_node_c.toml`

Reply:
I dont think these need to be tweaked by users, generally these should just work. These are here to make it clear what tweaks could be made to the protocol. But welcome to do that in another PR if you feel it will be valuable.